tests/python/README

```diff
@@ -0,0 +1,25 @@
+Issue 3227 (https://github.com/ispc/ispc/issues/3227) asked to explore Python bindings in ispc.
+
+This implementation uses the built-in ctypes module to show an example how to call ispc code
+from Python. Consider this a Proof Of Concept (PoC) with room for improvement.
+
+Design decisions:
+* Uniqify test function names in .ispc code to avoid name collision when generating the shared object.
+  This, along with removing the inlcude for test_static.cpp, necessitated creating a copy of the test
+  files. I don't like the redundnacy but I did not want to break functionality of existing tests. Future
+  efforts can clean this if needed.
+* One shared object is generated that contains all enabled tests. This reduces the number of dlls/
+  shared objects loaded during the Python run code. Per-test shared objects could be enabled if desired,
+  it just means a bit more Python code to generate the objects and load them.
```

Comment:
ISPC has > 1500 tests and I don't think we will be willing to describe each test in TestRunner.py. However ISPC uses only ~10 test signatures declared in https://github.com/ispc/ispc/blob/main/tests/test_static.isph. I think it makes sense to add support for all signatures from test_static.isph and generate a shared object for each test.
With this approach we would be able to run all the tests easily and you don't need to duplicate them.

Reply:
I’d expect the new test runner to be quite similar to `run_tests.py`, just without the `test_static.cpp` layer (something like top_bind.cpp should be fine). The issue with adding an extra config is that every new test would need to be manually added to the config, which won’t be practical until the process is fully automated and embedded in our practice. Also, handling a dozen test signatures feels much simpler to me than managing over a thousand tests. I see the advantages of your approach (and we may use it in a long term), but in the short term, I believe mirroring the `run_tests.py` logic is the better path forward.